### PR TITLE
Correct stop state by ignition.

### PR DESCRIPTION
**Pull request by Abyss777**
Here is the implementation of additional stop detection by ignition.

In case devices always report `ignition = false` made this function switchable.

### CODE REVIEW

File: src/org/traccar/Context.java [hunk 1]
```diff
@@ -256,7 +256,8 @@ public static TripsConfig initTripsConfig() {
                 config.getLong("report.trip.minimalTripDuration", 300) * 1000,
                 config.getLong("report.trip.minimalParkingDuration", 300) * 1000,
                 config.getBoolean("report.trip.greedyParking"),
-                config.getLong("report.trip.minimalNoDataDuration", 3600) * 1000);
+                config.getLong("report.trip.minimalNoDataDuration", 3600) * 1000,
+                config.getBoolean("report.trip.stopOnIgnitionOff"));
```
Comment (Abyss777) [inline src/org/traccar/Context.java]: Parameter name is controversial.

**Comment by tsmgeek** [comment]
Does stop detection also take note of the movement flag that some devices can report?

File: src/org/traccar/Context.java (hunk 1 above)
Comment (tananaev) [inline src/org/traccar/Context.java]: Can we just name it something like `report.trip.useIgnition`?

**Comment by Abyss777** [comment]
Yes, it uses `KEY_MOTION` attribute that reported by device or calculated automatically by speed if not.

**Comment by Abyss777** [comment]
Renamed parameter.

**Comment by tananaev** [comment]
Merged, thanks.

**Pull request merged**